lib/ansible/module_utils/mysql.py

```diff
@@ -75,6 +76,9 @@ def mysql_connect(module, login_user=None, login_password=None, config_file='',
 
     db_connection = mysql_driver.connect(**config)
     if cursor_class is not None:
-        return db_connection.cursor(cursorclass=mysql_driver.cursors.DictCursor)
+        if mysql_driver is sys.modules['pymysql']:
```

Comment:
Please don't do it this way. Set a flag after line 34 resp. 37 (the `import`s) and use that one here.

Reply:
Yes, you're right. That's ways better. :)